src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/NomisSyncRequest.kt

```diff
@@ -12,13 +12,11 @@ data class NomisSyncRequest(
     requiredMode = Schema.RequiredMode.NOT_REQUIRED,
     example = "123e4567-e89b-12d3-a456-426614174000",
     nullable = true,
-    defaultValue = "null",
   )
   val id: UUID? = null,
   @get:Schema(
     description = "Set to true for initial migration, omit or set to false for updates",
     requiredMode = Schema.RequiredMode.NOT_REQUIRED,
-    allowableValues = ["true"],
```

Comment:
this caused
```
    val initialMigration: NomisSyncRequest.InitialMigration? = false
```
to be generated

Reply:
new generation creates
```kotlin
    val initialMigration: kotlin.Boolean? = false
```
still a bit odd to have a `NOT_REQUIRED` field that isn't nullable in your code, but I've left that alone.

Reply:
didn't really make sense to limit the `allowableValues` like that anyway when you've got the `defaultValue` to be `false` 😁